src/components-examples/cdk-experimental/scrollable-table-body/cdk-scrollable-table-body-flex/cdk-scrollable-table-body-flex-example.css

```diff
@@ -0,0 +1,10 @@
+/**
+ * Add basic flex styling so that the cells evenly space themselves in the row.
+ */
+cdk-row, cdk-header-row, cdk-footer-row {
```

Comment:
It might be better to use the CSS classes for these selectors since they'll automatically work if we switch the example to use a `table`-based `mat-table`.

Reply:
+1, we prefer using css classes for styling over element selectors, and in examples we always prefix styled with `example-` to make it clear which styles come from the component and which come from the example.